packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,11 +6,25 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## December
 
+### Weekly release 2022.12.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                      |December 21, 2022            |
+
+#### Bugs fixed in 2022.12.2
+
+* UPDATE requests to the `/brands/{brandId}/pages/sign-in/preview` and `/brands/{brandId}/pages/sign-in` endpoints sometimes returned a `null` response that resulted in an HTTP 400 Bad Request error. (OKTA-551632)
+
+* If an admin added redirect URIs and reached the character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
+
+* When an admin viewed a user's profile, an error sometimes occurred. (OKTA-558220)
```

Comment:
Can we be more specific from dev perspective? This seems like an Admin Console bug? What's the API request?

Reply:
Confirmed with Susan that this is correct.